service/src/main/resources/logback.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--We need this file to override the terra-commons-lib configuration for human-readable logging-->
+<configuration scan="true" scanPeriod="30 seconds">
+
+    <variable name="LOG_PATH" value="logs"/>
+
+    <if condition='!isDefined("DRSHUB_LOG_APPENDER")'>
+        <then>
+            <variable name="DRSHUB_LOG_APPENDER" value="Console-Stackdriver" />
+        </then>
+    </if>
```

Comment:
> This also adds support for human-readable logging. When running locally, set DRSHUB_LOG_APPENDER=Console-Standard in your env, and DrsHub will log normal lines instead of Stackdriver JSON. Useful for local development!

That's awesome! Could we add this to the README?

Reply:
What README??? 🤣 I'll create a README  and add a logging section